content/updates/2021-04/1-Review-your-proposals-more-efficiently.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_img: ./img/revamped-proposal-screen.png
+pim_announcement_alt_img: "New proposals screen"
+pim_announcement_audience:
+- EE
+---
+
+# Review your proposals more efficiently
+::: meta-data type="Improvement" features="Productivity, Collaboration" available="late April" in="EE" link-to-doc="../articles/review-products-proposals.html"
+
+We're glad to introduce the revamp of our proposals screen. It's now even more practical and efficient for those of you who, depending on their catalog rights, need to review the enrichment work of teammates. 
+
+Among other improvements, you can notice that:
+- You now have, directly from within the proposal, a preview of your image assets, your PDFs, your YouTube links, and your reference entity records. A great time-saver when reviewing them!
+- Proposal layout is also now rearranged. Old and suggested new values are now easier to compare, making the whole process of values review more convenient.
```

Comment:
- The layout of the proposal has also been reorganized. Old and newly suggested values are now easier to compare, making the whole review process more convenient.

Reply:
Forgot to create a suggestion, Tata?
```suggestion
- The layout of the proposal has also been reorganized. Old and newly suggested values are now easier to compare, making the whole review process more convenient.
```